src/dbspro/cli/init.py

```diff
@@ -0,0 +1,90 @@
+"""
+Create and initialize a new analysis directory.
+"""
+import logging
+import os
+import os.path
+import sys
+import dnaio
+from pathlib import Path
+from importlib_resources import read_binary
+import pkg_resources
+
+logger = logging.getLogger(__name__)
+
+CONFIGURATION_FILE_NAME = "dbspro.yaml"
+ABC_FILE_NAME = "ABC-sequences.fasta"
+
+
+def add_arguments(parser):
+    parser.add_argument("reads", type=Path, help="Read file (.fastq.gz)")
+    parser.add_argument("directory", type=Path, help="New analysis directory to create")
+    parser.add_argument("--abc", default=pkg_resources.resource_filename("dbspro", "ABC-sequences.fasta"), type=str,
+                        metavar="ABC-sequences.fasta",
+                        help="Antibody barcode sequence (ABC) sequnce fasta file")
+
+    return parser
+
+
+def main(args):
+    init(args.directory, args.reads, args.abc)
+
+
+def init(directory: Path, reads: Path, abc: str):
+    if " " in str(directory):
+        logger.error("The name of the analysis directory must not contain spaces")
+        sys.exit(1)
+
+    create_and_populate_analysis_directory(directory, reads, abc)
+
+    logger.info(f"Directory {directory} initialized.")
+    logger.info(
+        'Edit %s/%s and %s/%s, then run "cd %s && dbspro run" to start the analysis',
+        directory,
+        CONFIGURATION_FILE_NAME,
+        directory,
+        ABC_FILE_NAME,
+        directory,
+    )
```

Comment:
Change this info message. The `ABC_FILE_NAME` file does not have to be edited after running the `init`.

Reply:
My thought was that this should be edited if you have other ABCs, since this also is the reason why we have it in a fastq file rather than specifying the sequences in the config file. I could change the phrasing to more clear that this is if you are running custom ABC seqs though.

Reply:
I realized it's defined by args so I'll remove it from the message